packages/secure-store/kdf.ts

```diff
@@ -10,12 +10,14 @@ import argon2 from 'react-native-argon2'
  *
  */
 const derive = async (pin: string, salt: string): Promise<string> => {
+  // Takes about 1.5 second on iPhone 14 Pro
+  // Need to test on Android/different devices as well
   const { rawHash } = await argon2(pin, salt, {
     hashLength: 32,
     mode: 'argon2id',
     parallelism: 4,
-    iterations: 1,
-    memory: 21,
+    iterations: 8,
```

Comment:
Why did the values change? Did we not receive these form a spec?

Reply:
react-native-argon2 would crash. Did you test it with the previous values? This now takes around 1 sec on my phone. Wanted to do another check later to make it work also on android / older devices